### PR TITLE
Fix typos and add some punctuation #76

### LINKED ISSUE #76: Found some typos in wsl-boostrap-2020 blog post

### PR DESCRIPTION
**Pull request by gbgabiola**
Aside from fixing typos and adding punction, I also remove "this time I'll need to move the created .ssh folder to" from line 319, which is cut or broken and I think it's duplicated on line around 514.

Fix #76

### CODE REVIEW

**Comment by gbgabiola** [comment]
Fix #76

File: content/posts/2020/08/26/wsl-bootstrap-2020/index.mdx [hunk 1]
```diff
@@ -33,8 +33,9 @@ the past:
 - [Windows Subsystem Linux setup]
 - [Windows Web-dev bootstrap]
 
-This time round it was just as confusing (for me) at the times before,
-so this is why I'm making notes for future Scott to refer to. 🔥
+This time 'round it was just as confusing (for me) at the times
```
Comment (spences10) [inline content/posts/2020/08/26/wsl-bootstrap-2020/index.mdx]: Remove this change, I don't use this punctuation

File: content/posts/2020/08/26/wsl-bootstrap-2020/index.mdx [hunk 2]
```diff
@@ -78,8 +79,8 @@ instructions from the [Windows WSL Install] docs.
 You need to be on Windows version 1903 or higher, Build 18362 or
 higher.
 
-If you don't have Windows Pro you should be able to enable WSL 2 on
-[Windows 10 home].
+If you don't have Windows Pro you should still be able to enable WSL 2
```
Comment (spences10) [inline content/posts/2020/08/26/wsl-bootstrap-2020/index.mdx]: Unnecessary word, remove this please

File: content/posts/2020/08/26/wsl-bootstrap-2020/index.mdx [hunk 3]
```diff
@@ -246,8 +247,8 @@ dev are VSCode and **WSLGit**.
 WSLGit being the important one that enables the use of Git installed
 on my WSL instance to work in VSCode on Windows.
 
-Prior to this I [had to install Git on Windows] which had it's own set
-of problems with line endings and file permissions.
+Prior to this, I [had to install Git on Windows] which had it's own
+set of problems with line endings and file permissions.
```
Comment (spences10) [inline content/posts/2020/08/26/wsl-bootstrap-2020/index.mdx]: Unnecessary punctuation, remove this please

File: content/posts/2020/08/26/wsl-bootstrap-2020/index.mdx [hunk 4]
```diff
@@ -315,8 +316,7 @@ omf install nvm
 
 Now I can add my aliases to fish, I have a list that I always use from
 my [Cheat Sheets] site. This is shortening things like `git` to `g`
-(yes) and `yarn add` to `ya`. this time I'll need to move the created
-`.ssh` folder to
+(yes) and `yarn add` to `ya`.
```
Comment (spences10) [inline content/posts/2020/08/26/wsl-bootstrap-2020/index.mdx:318]: Thanks for this, much appreciated, multi cursor fail!

File: content/posts/2020/08/26/wsl-bootstrap-2020/index.mdx [hunk 5]
```diff
@@ -396,7 +396,7 @@ a try:
 For fnm you may need `unzip` for if not installed already:
 
 ```bash
-sudo apt innstall unzip
+sudo apt install unzip
```
Comment (spences10) [inline content/posts/2020/08/26/wsl-bootstrap-2020/index.mdx:398]: Thanks for this 🙏

**Review by spences10 — CHANGES_REQUESTED** [review]
Thanks for this @genesisgabiola, please refer to my comments on parts I'm not willing to put in there and thanks for the typos.

**Comment by spences10** [comment]
Awesome! Thanks you so much @genesisgabiola 🙏

**Comment by gbgabiola** [comment]
> Awesome! Thanks you so much @genesisgabiola 🙏

I'm happy to contribute to your project 👍

**Comment by genesisgabiola** [comment]
fix #76

**Pull request merged**